### PR TITLE
UAR-1454 Update banner URL for Remove

### PR DESCRIPTION
**Pull request by dlloyd1-ch**
### JIRA link

https://companieshouse.atlassian.net/browse/UAR-1454

### Change description

- Update the URL banner for Remove to be the Remove guidance page on the UK Gov website.
- Update the Remove URL pages in the README

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars [hunk 1]
```diff
@@ -33,7 +33,7 @@ landing_page_url = "https://www.gov.uk/guidance/register-an-overseas-entity"
 landing_page_starting_new_url = "/register-an-overseas-entity/starting-new"
 payment_fee = "100"
 update_landing_page_url = "/update-an-overseas-entity/continue-with-saved-filing"
-remove_landing_page_url = "TO BE DEFINED"
+remove_landing_page_url = "https://www.gov.uk/guidance/remove-an-overseas-entity"
```
Comment (mwestacott) [inline terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars:36]: Looks fine. The URL gives a 'Page not found' message at the moment. Assume it won't be LIVE until Wednesday.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,8 @@
 
 Web front-end for the **register an overseas entity and tell us about its beneficial owners** service. Link to the live page [here](https://www.gov.uk/guidance/register-an-overseas-entity)
 </br>
-
+Web front-end for the **apply to remove an overseas entity from the register** service. Link to the live page [here](https://www.gov.uk/guidance/remove-an-overseas-entity)
```
[the remark this answers is not in the document]
Comment (dlloyd1-ch) [reply README.md:7]: Found the Update live URL so have added it here.

File: terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars (hunk 1 above)
Comment (dlloyd1-ch) [reply terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars:36]: That is the assumpton.

Comment (makhtar-ch) [reply terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars:36]: Maybe not necessary, but would we need to change the cidev and staging value for remove_landing_page_url?

**Review by markpit — APPROVED** [review]

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (mwestacott) [reply README.md:7]: Thanks.

File: terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars (hunk 1 above)
Comment (mwestacott) [reply terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars:36]: I think that they are probably fine. We don't have a static GOV UK page for those envs, hence why they link somewhere different.

**Review by mwestacott — APPROVED** [review]

**Pull request merged**